anonymizor/anonymizor.py

```diff
@@ -9,12 +9,41 @@
 import time
 import yaml
 
-from detect_secrets.plugins import keyword
 from password_strength import PasswordStats
 
-
 logger = logging.getLogger(__name__)
 
+DENYLIST = (
+    'api_?key',
+    'auth_?key',
+    'service_?key',
+    'account_?key',
+    'db_?key',
+    'database_?key',
+    'priv_?key',
+    'private_?key',
+    'client_?key',
+    'db_?pass',
+    'database_?pass',
+    'key_?pass',
+    'key_?data',
+    'password',
+    'passwd',
+    'pass',
+    'pwd',
+    'secret',
+    'contraseña',
+    'contrasena',
+)
+# Includes ], ', " as closing
+CLOSING = r'[]\'"]{0,2}'
+AFFIX_REGEX = r'\w*'
+DENYLIST_REGEX = r'|'.join(DENYLIST)
+# Support for suffix after keyword i.e. password_secure = "value"
+DENYLIST_REGEX = r'({denylist}){suffix}'.format(
+    denylist=DENYLIST_REGEX,
+    suffix=AFFIX_REGEX,
+)
 
 def gen_sample_password():
```

Comment:
@goneri As we'll be replacing the secrets with `"{{ }}"`, we can remove this fn. as it's not needed